index.html

```
@@ -951,50 +951,62 @@ <h3 class="font-headline font-extrabold text-lg sm:text-xl">2026 Timeline</h3>
+      </span>
+    </div>
+  </div>
+
+  <div class="flex flex-wrap items-center justify-between gap-4 mb-8">
     <p class="font-label text-xs uppercase tracking-widest text-zinc-500">Showing <strong id="orgCount">184</strong> of 184 organizations</p>
+<<<<<<< HEAD
     <div class="flex items-center gap-4">
       <select id="categoryFilter" aria-label="Filter by category" class="bg-surface-container-low border-none rounded-xl text-xs font-bold focus:ring-2 focus:ring-zinc-500 focus:outline-none cursor-pointer text-zinc-600">
+=======
+    <div class="flex flex-wrap items-center gap-4">
+      <select id="categoryFilter" class="bg-surface-container-low border-none rounded-xl text-xs font-bold focus:ring-2 focus:ring-zinc-500 focus:outline-none cursor-pointer text-zinc-600">
```

HTMLHint (1.9.2)

[error] The id value [ categoryFilter ] must be unique.

(id-unique)